examples/gds/client.c

```diff
@@ -0,0 +1,126 @@
+/* This work is licensed under a Creative Commons CCZero 1.0 Universal License.
+ * See http://creativecommons.org/publicdomain/zero/1.0/ for more information. */
+/*
+ * A simple server instance which registers with the discovery server (see server_lds.c).
+ * Before shutdown it has to unregister itself.
+ */
+
+#include "open62541.h"
+
+UA_Boolean running = true;
+
+static void stopHandler(int sig) {
+    UA_LOG_INFO(UA_Log_Stdout, UA_LOGCATEGORY_USERLAND, "received ctrl-c");
+    running = false;
+}
+
+int main(int argc, char **argv) {
+    signal(SIGINT, stopHandler);
+    signal(SIGTERM, stopHandler);
+
+    // Initialize the client and connect to the GDS
+    UA_ClientConfig config = UA_ClientConfig_default;
+    UA_String applicationUri = UA_String_fromChars("urn:open62541.example.server_register");
+    UA_Client *client = UA_Client_new(config);
+    UA_StatusCode retval = UA_Client_connect_username(client, "opc.tcp://localhost:4841", "user1", "password");
```

Comment:
Not so important, but why don't you use the default 4840 port? Is GDS supposed to be on 4841?